Automattic/newspack-plugin
# Add Site Design Wizard and link it to the Setup

### All Submissions:






### Changes proposed in this Pull Request:

Use the Theme Selection as the Site Design Wizard to match Theme Selection step during setup.

During the setup we now import `ThemeSelection` from this new wizard so we don't duplicate the code.

![Screen Recording 2020-02-04 at 14 50 46](https://user-images.githubusercontent.com/177929/73755934-b527be00-475e-11ea-9724-ccb0b7a9776b.gif)

### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Navigation to Dashboard > Site Design (or Sidebar menu > Site Design)
3. Try to activate all the themes and check if it worked on the front-end
4. Go through the Setup and try to activate the themes during the Theme Selection step

### Other information:
